physx/src/scene.rs

```diff
@@ -88,6 +91,39 @@ impl Scene {
         })
     }
 
+    pub fn add_capsule_controller(
+        &mut self,
+        material: *mut PxMaterial,
+        height: f32,
+        radius: f32,
+        step_offset: f32,
+    ) -> Controller {
+        unsafe {
+            if self.controller_manager.is_none() {
```

Comment:
I think it makes more sense to put this as part of the scene builder API, not as a runtime opt-in thing. That also lets the user decide whether to use locking.

Reply:
Also, does this require any extensions to be loaded?

Reply:
Sure, that makes sense. Would still you leave it as optional? 
And no, this doesn't require any extensions.

Reply:
I think having it as optional makes sense, as there's lots of use-cases where you probably don't use it. Should be fine as a `use_controller_manager(enable: bool, locking: bool)` on the builder.